docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,90 @@
+# Sensor Models
+
+This document is intended to provide information related to sensor models by presenting an introduction to sensor models, a brief explanation of the Community Sensor Model (CSM), and an overview of several ASC software packages that directly relate to the instantiation and exploitation of sensor models.
+
+## Introduction to Sensor Models
+
+### Reference Systems and Reference Frames
+
```

Comment:
It is weird to tell use what this is about and then start with 'before we do that, here is a fair bit of text'. Suggest splitting into a different document and identifying as a pre-requisite.